backend/packages/Upgrade/src/api/services/FeatureFlagService.ts

```diff
@@ -351,6 +359,27 @@ export class FeatureFlagService {
         updatedAt,
         ...flagDoc
       } = flag;
+
+      if (oldFlagDoc.context[0] !== flagDoc.context[0]) {
+        // Create delete audit logs for inclusion and exclusion lists
+        if (includeListIds.length) {
```

Comment:
You should use transaction to add the audit logs. Use `Promise.all` for all the three operations

Reply:
Add Promise.all for all the requests